docs/docs/en/guide/installation/pseudo-cluster.md

```diff
@@ -115,26 +97,47 @@ deployUser="dolphinscheduler"
 
 ### Modify `dolphinscheduler_env.sh`
 
-File `dolphinscheduler_env.sh` describes the database configuration of DolphinScheduler, which in the path `bin/env/dolphinscheduler_env.sh`
-and some tasks which need external dependencies or libraries such as `JAVA_HOME` and `SPARK_HOME`. You could ignore the
-task external dependencies if you do not use those tasks, but you have to change `JAVA_HOME`, registry center and database
+File  `./bin/env/dolphinscheduler_env.sh` describes the following configurations:
+* Database configuration of DolphinScheduler, see [Initialize the Database](#initialize-the-database) for detailed instructions.
```

Comment:
Add one more blank line will be more confortable
```suggestion

* Database configuration of DolphinScheduler, see [Initialize the Database](#initialize-the-database) for detailed instructions.
```

Reply:
Same as ZH docs

Reply:
Sure I will fix it.